src/state/dashboard/dashboard.epics.ts

```diff
@@ -0,0 +1,21 @@
+import { catchError, filter, map, of, switchMap } from 'rxjs'
+import { Epic, combineEpics } from 'redux-observable'
+
+import { http } from 'api/http'
+import { dashboardActions as actions } from 'state/dashboard'
+
+const getStatistics$: Epic = (actions$, state$) =>
+  actions$.pipe(
+    filter(actions.fetchData.match),
+    switchMap(() =>
+      http()
+        .get('/organisations/statistics')
+        .pipe(
+          map((response) => actions.fetchDataSuccess(response.data)),
+          catchError((error) => of(actions.fetchDataError(error))),
+        ),
+    ),
+    // handleEpicError(actions.fetchDataError, 'Failed to get statistics'),
```

Comment:
should it be commented?

Reply:
Yes. Error handler is not yet implemented.